libs/utils/generateGlobalProps.ts

```diff
@@ -1,12 +1,11 @@
 import { Api } from "@cennznet/api";
 import { SectionUri } from "@/libs/types";
+import { CENNZ_NETWORK } from "@/libs/constants";
 
 export interface GlobalProps {}
 
 export default async function generateGlobalProps(
 	section: SectionUri
 ): Promise<GlobalProps> {
-	const api = await Api.create({ provider: process.env.NEXT_PUBLIC_API_URL });
-
 	return {};
 }
```

Comment:
maybe delete this file, or use it to define props per page instead of calling Api.create() on each page?

Reply:
good call, let's remove it